demisto_sdk/commands/create_artifacts/content_artifacts_creator.py

```diff
@@ -1602,14 +1602,7 @@ def dump_link_files(
                 new_created_files.append(new_file)
     # Handle case where object first time dump.
     else:
-        target = dest_dir / content_object.normalize_file_name()
-        if (
-            target.exists()
-            and target.stat().st_mtime >= artifact_manager.execution_start
-        ):
-            raise DuplicateFiles(target, content_object.path)
-        else:
-            new_created_files.extend(content_object.dump(dest_dir=dest_dir))
+        new_created_files.extend(content_object.dump(dest_dir=dest_dir))
```

Comment:
@dorschw  why did you change the logic here?

Reply:
@ilaner why did you change the logic here?

Reply:
A test failed on it. Need to revert and investigate the failures.

Reply:
fixed in #3645